varc_core/systems/linux.py

```diff
@@ -107,9 +108,21 @@ def dump_processes(self) -> None:
                             for map in maps:
                                 page_start = map[0]
                                 page_len = map[1] - map[0]
-                                mem_page_content = self.read_bytes(pid, page_start, page_len)
-                                if mem_page_content:
-                                    tmpfile.write(mem_page_content)
+                                if page_len > self._MAX_VIRTUAL_PAGE_CHUNK:
```

Comment:
I don't think you need the if, just set `page_len = min(page_len, self._MAX_VIRTUAL_PAGE_CHUNK)`

Reply:
Fwiw given how we then treat memory (via ripraw/carving) inconsistent memory may not be such an issue